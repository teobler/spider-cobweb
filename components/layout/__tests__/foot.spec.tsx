import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../foot';

describe('<Footer />', () => {
  it('should render a empty footer', () => {
    const layoutJson = renderer.create(<Footer />).toJSON();
    expect(layoutJson).toMatchSnapshot();
  });
});
