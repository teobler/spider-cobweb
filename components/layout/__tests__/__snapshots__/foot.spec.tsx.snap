// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render a empty footer 1`] = `
<footer
  className="spider-layout-footer"
/>
`;
